Lombiq.ChartJs.Tests.UI/Lombiq.ChartJs.Tests.UI.csproj

```diff
@@ -29,8 +29,12 @@
     <ProjectReference Include="..\..\..\..\test\Lombiq.UITestingToolbox\Lombiq.Tests.UI\Lombiq.Tests.UI.csproj" />
   </ItemGroup>
 
+  <PropertyGroup Condition="'$(NuGetBuild)' == 'true'">
+    <DebugType>embedded</DebugType>
+  </PropertyGroup>
```

Comment:
Are you sure we need this in the other project too?

Reply:
When the `DebugType`is set to `embedded` no `pdb` files are generated(see: https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-options/code-generation#:~:text=Emit%20debugging%20information%20into%20the%20.dll/.exe%20itself%20(.pdb%20file%20is%20not%20produced)%20using) which results in an empty `snupkg` and finally the `nuget push` exits with `error: Response status code does not indicate success: 400 (The package does not contain any symbol (.pdb) files.).` See: https://github.com/Lombiq/Orchard-Chart.js/actions/runs/5229699167/jobs/9442833843#step:4:2875
To avoid this error, I set `dotnet-pack-include-symbols: "false"`, but in this case, the debug info is not published for the other packages, this is why I added this configuration to other projects too.

Reply:
I see, so the other projects aren't actually needed to be `DebugType = embedded`, but we can't mix these.

Is the package possible to debug in NuGet with this config, like can you step into the methods of the package?

BTW I see no relevant change in the `publish-nuget` action since Chart.js v4.0.0 and it being included in the NuGetTest solution, so I don't know what has caused this change in behavior.

Reply:
> I see, so the other projects aren't actually needed to be DebugType = embedded, but we can't mix these.

Exactly

> Is the package possible to debug in NuGet with this config, like can you step into the methods of the package?

Yes

![image](https://github.com/Lombiq/Orchard-Chart.js/assets/15349881/24b768f5-2137-4c7d-8afa-02154abb1a8f)

> BTW I see no relevant change in the publish-nuget action since Chart.js v4.0.0 and it being included in the NuGetTest solution, so I don't know what has caused this change in behavior.

I didn't find any relevant changes as well.

Reply:
Great, then!